binding_generator.py

```diff
@@ -717,9 +746,11 @@ def get_icall_name(sig):
 
 
 
-def get_used_classes(c):
+def get_used_classes(c, use_template_get_node):
     classes = []
     for method in c["methods"]:
+        if use_template_get_node:
```

Comment:
This function gets all classes a given class depends on. Why does this require to change method name here? We would only need to do the correction if we actually need to write the method in a file.

In fact, perhaps you did not need to put `correct_method` everywhere, just do it once on the `Node` class after getting all classes in the beginning?